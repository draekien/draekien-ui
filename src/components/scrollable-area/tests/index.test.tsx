import React from 'react';
import { render } from '@testing-library/react';
import ScrollableArea from '..';

describe('ScrollableArea component', () => {
  test('should match snapshot', () => {
    const { container } = render(
      <ScrollableArea maxHeight="200px">
        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Aliquid nulla
        dolor fuga, voluptate autem cupiditate enim quae beatae repellendus
        velit quisquam quam tempora, commodi dolorum. Eos eaque consequatur
        doloremque officia.
      </ScrollableArea>
    );

    expect(container).toMatchSnapshot();
  });
});
