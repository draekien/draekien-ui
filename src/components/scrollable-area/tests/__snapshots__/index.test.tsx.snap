// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollableArea component should match snapshot 1`] = `
<div>
  <div
    style="position: relative; overflow: hidden; width: 100%; height: 100%;"
  >
    <div
      style="position: absolute; top: 0px; left: 0px; right: 0px; bottom: 0px; overflow: scroll; margin-right: 0px; margin-bottom: 0px;"
    >
      <div
        style="max-height: 200px;"
      >
        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Aliquid nulla dolor fuga, voluptate autem cupiditate enim quae beatae repellendus velit quisquam quam tempora, commodi dolorum. Eos eaque consequatur doloremque officia.
      </div>
    </div>
    <div
      style="position: absolute; height: 6px; display: none; right: 2px; bottom: 2px; left: 2px; border-radius: 3px;"
    >
      <div
        style="position: relative; display: block; height: 100%; cursor: pointer; border-radius: inherit; background-color: rgba(0, 0, 0, 0.2);"
      />
    </div>
    <div
      style="position: absolute; width: 6px; display: none; right: 2px; bottom: 2px; top: 2px; border-radius: 3px;"
    >
      <div
        style="position: relative; display: block; width: 100%; cursor: pointer; border-radius: inherit; background-color: rgba(0, 0, 0, 0.2);"
      />
    </div>
  </div>
</div>
`;
